LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2023 Catamorphic, Co.
```

Comment:
Normally I don't change these dates, but we are publishing it now, so it seemed reasonable.